### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by Terminateit**

### CODE REVIEW

File: tests/scripts/FKtest.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        
+    def test_FK_nullneg(self):
+        L = np.array([0, 0, 0, 0, 0, 0])
+        q = np.array([-pi/6, -pi/4, -pi, -pi/3])
+        self.assertEqual(DK.DirectKinematics(q, L), None)
+        
+    def test_FK_nullpos(self):
+        L = np.array([0, 0, 0, 0, 0, 0])
+        q = np.array([pi/4, pi/3, pi/2, pi/3])
+        self.assertEqual(DK.DirectKinematics(q, L),None)
```
Comment (l1va) [inline tests/scripts/FKtest.py]: cool, i like this test file.

File: tests/scripts/subscriber.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+def main():
+	counter = 0
+	rospy.init_node('image_listener')
+
+	#topics for each link
+
+	link1_topic = '/crobot/joint0_position_controller/state'
+	link2_topic = '/crobot/joint1_position_controller/state'
+	link3_topic = '/crobot/joint2_position_controller/state'
+	link4_topic = '/crobot/joint3_position_controller/state'
```
Comment (l1va) [inline tests/scripts/subscriber.py]: it is not cool. Just copy of Ramil's tests.

Comment (Terminateit) [reply tests/scripts/subscriber.py]: I didn't copy it. The marks are unfair

File: atlas_launch/scripts/control.py [hunk 3]
```diff
@@ -19,12 +17,16 @@ def rotate1(joint, min, max):
 
 if __name__ == '__main__':
 	time.sleep(5)
-	rospy.init_node('rotate', anonymous=True)
+	rospy.init_node('rotate', anonymous=True)	
+	pub = []	
+	for i in range(4):
+		pub.append(rospy.Publisher('/crobot/joint'+str(i+1)+'_position_controller/command', Float64, queue_size=10))
```
[the remark this answers is not in the document]
Comment (Terminateit) [reply atlas_launch/scripts/control.py]: I think, it doesn't matter

**Pull request merged**